### PR TITLE
No longer index trashed blocks.

### PR DESCRIPTION
**Pull request by maethu**

### CODE REVIEW

File: ftw/simplelayout/tests/test_trash.py [hunk 1]
```diff
@@ -39,3 +40,13 @@ def test_requires_modify_portal_content_permission_for_restoring_blocks(self):
...
+
+    def test_dont_index_trashed_blocks(self):
+        portal = api.portal.get()
+        page = create(Builder('sample container'))
+        block = create(Builder('sample block').titled(u'qwertz').within(page))
+
+        self.assertEquals(1, len(portal.portal_catalog(SearchableText='qwertz')))
+
+        Trasher(block).trash()
+        self.assertEquals(1, len(portal.portal_catalog(SearchableText='qwertz')))
```
Comment (jone) [inline ftw/simplelayout/tests/test_trash.py]: Shouldnt that be 0 then? 🤔

Comment (maethu) [reply ftw/simplelayout/tests/test_trash.py]: ugh

**Comment by maethu** [comment]
@jone Pls check again... I was probably kinda sleeping. Now it works as expected.

File: ftw/simplelayout/configure.zcml [hunk 2]
```diff
@@ -114,4 +114,16 @@
         handler=".handlers.modify_parent_on_block_edit"
         />
 
+    <subscriber
+        for="ftw.simplelayout.interfaces.ISimplelayoutBlock
+             ftw.trash.interfaces.IObjectTrashedEvent"
```
Comment (jone) [inline ftw/simplelayout/configure.zcml]: This makes `ftw.trash` required for simplelayout, breaking tests -> https://ci.4teamwork.ch/builds/208737/tasks/343649

**Pull request merged**